UnitySDK/Assets/ML-Agents/Scripts/BroadcastHub.cs

```diff
@@ -11,50 +11,27 @@ namespace MLAgents
     [System.Serializable]
```

Comment:
is there any need for this class anymore?

Reply:
Yes. Still needs more work to kill it completely. You must put your Brains into the Hub so the Academy knows what brains are training.